django/db/models/indexes.py

```diff
@@ -4,12 +4,11 @@
 
 __all__ = ['Index']
 
-# The max length of the names of the indexes (restricted to 30 due to Oracle)
-MAX_NAME_LENGTH = 30
-
 
 class Index:
     suffix = 'idx'
+    # Default max length of the names of the indexes (restricted to 30 due to Oracle)
+    max_name_length = 30
```

Comment:
The idea is to limit the length for indexes that might be used across different databases. If a third-party app author is using PostgreSQL, for example, they should create index names <= 30 so that the app is usable on Oracle. The indexes in `contrib.postgres` don't need this limit since they are only usable on PostgreSQL.